ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/support/commands.js

```diff
@@ -541,4 +541,10 @@ Cypress.Commands.add('assignUser', () => {
             cy.get('[class="govuk-button"]').click()
         }
     })
-})
\ No newline at end of file
+})
+
+//Navigate To Filter Projects section
+Cypress.Commands.add('navigateToFilterProjects',() => {  
+  cy.get('.govuk-details__summary-text').click();
```

Comment:
If we can avoid it I think we should be using id's or test id's rather than classes.